### ISSUE TITLE
NubilesNET Keyword no longer matching?

### ISSUE THREAD

**Issue by covrdinbeez** [PAhelper/PhoenixAdult.bundle#1175]
**Describe the bug**
Per the sitelist, I used to be able to match Nubiles.net (NubilesNet) movies using the site name "NubilesNet" but it doesn't seem to be working anymore. The scraper seems to identify the proper site for matching (Nubiles) but then adds the NET to the title (sceneID) and nothing matches. That seems odd. (Using just Nubiles still seems to work though.) Did I miss a change?

**Include**
1. Scene URL - https://nubiles.net/video/watch/107538/sugar-and-spice
2. File Name - NubilesNET.107538.-.Sugar.And.Spice.-.Liza.Smoke.-.20.08.15.XXX.1080p.HEVC.x265-PRT.mp4
     Search String - NubilesNET 107538
3. Log File
```
2021-01-26 12:14:55,755 (70000df85000) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcyMjQ2OHMyCmlkczE3Ck51YmlsZXNORVQgMTA3NTM4czQKbmFtZXM0CjIwMjBzNAp5ZWFycjAK
2021-01-26 12:14:55,755 (70000df85000) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2021-01-26 12:14:55,756 (70000df85000) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcyMjQ2OHMyCmlkczE3Ck51YmlsZXNORVQgMTA3NTM4czQKbmFtZXM0CjIwMjBzNAp5ZWFycjAK
2021-01-26 12:14:55,756 (70000df85000) :  INFO (agentkit:961) - Searching for matches for {'year': '2020', 'id': '722468', 'name': 'NubilesNET 107538'}
2021-01-26 12:14:55,756 (70000df85000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/722468/tree'
2021-01-26 12:14:55,760 (70000df85000) :  INFO (__init__:62) - ***MEDIA TITLE [from media.name]*** NubilesNET 107538
2021-01-26 12:14:55,760 (70000df85000) :  INFO (PAsearchSites:52) - mediaTitle w/ possible abbreviation: NubilesNET 107538
2021-01-26 12:14:55,766 (70000df85000) :  INFO (PAsearchSites:60) - mediaTitle w/ possible abbrieviation fixed: NubilesNET 107538
2021-01-26 12:14:55,770 (70000df85000) :  INFO (PAsearchSites:45) - Site found
2021-01-26 12:14:55,770 (70000df85000) :  INFO (PAsearchSites:72) - ^^^^^^^ siteNum: 540
2021-01-26 12:14:55,770 (70000df85000) :  INFO (PAsearchSites:73) - ^^^^^^^ Shortening Title
2021-01-26 12:14:55,771 (70000df85000) :  INFO (PAsearchSites:97) - Search Title (before date processing): NET107538
2021-01-26 12:14:55,771 (70000df85000) :  INFO (__init__:64) - {'siteName': 'nubiles', 'searchDate': None, 'siteNum': 540, 'searchTitle': 'NET107538'}
2021-01-26 12:14:55,772 (70000df85000) :  INFO (PAsearchData:17) - SearchData.title: NET107538
2021-01-26 12:14:55,772 (70000df85000) :  INFO (__init__:92) - Provider: networkNubiles
2021-01-26 12:14:55,772 (70000df85000) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2021-01-26 12:14:55,772 (70000df85000) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
```

**Comment by DirtyRacer1337** [comment]
`NubilesNET` not in list, you should use just `Nubiles`, anyway your filename works for me (because it have date). You also find a bug, I fixed it in 63257dd33948683ffccc1458209493e03f1fca39

**Comment by covrdinbeez** [comment]
Ok. Nubiles works. I could swear NubilesNet used to work, but maybe it was because I had the date in there and I no longer am using that naming scheme. Thanks.
